catalog/view/stylesheet/securesubmit.css

```diff
@@ -41,11 +52,13 @@
 
 #hpsPaymentForm .ss-label {
     text-transform:uppercase;
+    position: relative;
+    left: 1%;
 }
 
 #hpsPaymentForm .form-horizontal .control-label {
     text-align:left;
-    width:60%; 
+    width:60%;
     padding-left:none;
```

Comment:
Not an issue you introduced, but can you fix this? ['none' isn't a value of padding-left](https://developer.mozilla.org/en-US/docs/Web/CSS/padding-left)